bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/hbasesnapshots/ImportJobFromHbaseSnapshot.java

```diff
@@ -87,6 +97,18 @@ public static void main(String[] args) throws Exception {
     ImportOptions opts =
         PipelineOptionsFactory.fromArgs(args).withValidation().as(ImportOptions.class);
 
+    if(opts.getEnableSnappy() != null && opts.getEnableSnappy()) {
+        DataflowPipelineOptions dataFlowOpts = opts.as(DataflowPipelineOptions.class);
+        dataFlowOpts.setSdkContainerImage(CONTAINER_IMAGE_PATH_PREFIX + ReleaseInfo.getReleaseInfo().getVersion());
```

Comment:
Is there a way to fail fast here? Can we check if the image exists at this point?